packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/multiplier_stop_loss.js

```diff
@@ -0,0 +1,91 @@
+import { localize } from '@deriv/translations';
+import { getCurrencyDisplayCode } from '@deriv/shared';
+import { config } from '../../../../constants/config';
```

Comment:
We can give `constants` an alias.

Reply:
We don't have absolute imports in `bot-skeleton`  at the moment, but yeah it'll be better to have it.